frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/vmi-users-alert.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Alert, Button } from '@patternfly/react-core';
+import { history } from '@console/internal/components/utils';
+import { getName, getNamespace } from '@console/shared/src/selectors/common';
+import { useGuestAgentInfo } from '../../../hooks/use-guest-agent-info';
+import { GuestAgentInfoWrapper } from '../../../k8s/wrapper/vm/guest-agent-info/guest-agent-info-wrapper';
+import { VMIKind } from '../../../types';
+
+import './vmi-users-alert.scss';
+
+type VMIUsersAlertProps = {
+  vmi: VMIKind;
+  alertTitle?: string;
+  cancel: () => void;
+};
+
+export const VMIUsersAlert: React.FC<VMIUsersAlertProps> = ({ vmi, cancel, alertTitle }) => {
+  const [guestAgentInfoRaw] = useGuestAgentInfo({ vmi });
+  const guestAgentInfo = new GuestAgentInfoWrapper(guestAgentInfoRaw);
+  const userListLength = guestAgentInfo.getUserList()?.length || 0;
+
+  const name = getName(vmi);
+  const namespace = getNamespace(vmi);
+  const alertHref = `/k8s/ns/${namespace}/virtualmachines/${name}/details#logged-in-users`;
```

Comment:
what about standalone VMIs?

Reply:
Thanks !!! wow, no I remember why I took it out to be a separate modal !